### PR TITLE
feat: update frontend for next release

### PR DESCRIPTION
**Pull request by shekhar-shubhendu**
# Important
This PR also changes the `ctez.mligo` Please check the change here: https://github.com/tzConnectBerlin/ctez/blob/next/ctez.mligo#L211

The next version is deployed at: https://ctez-next.vercel.app/

### CODE REVIEW

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx [hunk 1]
```diff
@@ -48,10 +48,10 @@ const AddLiquidityComponent: React.FC<WithTranslation> = ({ t }) => {
 
   const calcMaxToken = (slippage: number, cashDeposited: number) => {
     if (cfmmStorage) {
-      const { tokenPool, cashPool } = cfmmStorage;
+      const { tezPool, cashPool } = cfmmStorage;
       const cash = cashDeposited * 1e6;
       const max =
-        Math.ceil(((cash * tokenPool.toNumber()) / cashPool.toNumber()) * (1 + slippage * 0.01)) /
+        Math.ceil(((cash * cashPool.toNumber()) / tezPool.toNumber()) * (1 + slippage * 0.01)) /
```
Comment (murbard) [inline frontend/app/src/pages/BuySell/AddLiquidity.tsx:54]: This makes the wrong assumption about price

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx [hunk 2]
```diff
@@ -48,10 +48,10 @@ const AddLiquidityComponent: React.FC<WithTranslation> = ({ t }) => {
 
   const calcMaxToken = (slippage: number, cashDeposited: number) => {
     if (cfmmStorage) {
-      const { tokenPool, cashPool } = cfmmStorage;
+      const { tezPool, cashPool } = cfmmStorage;
```
Comment (murbard) [inline frontend/app/src/pages/BuySell/AddLiquidity.tsx:51]: are you sure about the order?

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx [hunk 3]
```diff
@@ -61,10 +61,10 @@ const AddLiquidityComponent: React.FC<WithTranslation> = ({ t }) => {
 
   const calcMinPoolPercent = (slippage: number, cashDeposited: number) => {
     if (cfmmStorage) {
-      const { cashPool, lqtTotal } = cfmmStorage;
+      const { tezPool, lqtTotal } = cfmmStorage;
       const cash = cashDeposited * 1e6;
       const minLQTMinted =
-        ((cash * lqtTotal.toNumber()) / cashPool.toNumber()) * (1 - slippage * 0.01);
+        ((cash * lqtTotal.toNumber()) / tezPool.toNumber()) * (1 - slippage * 0.01);
```
Comment (murbard) [inline frontend/app/src/pages/BuySell/AddLiquidity.tsx:67]: wrong price calculation

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx (hunk 1 above)
Comment (murbard) [reply frontend/app/src/pages/BuySell/AddLiquidity.tsx:54]: you need to use the functions that compute what the actual price obtained is, using Newton descent etc

File: frontend/app/src/pages/BuySell/Conversion.tsx [hunk 4]
```diff
@@ -53,15 +54,28 @@ const ConvertComponent: React.FC<ConversionParams> = ({ t, formType }) => {
...
+              tezPool.toNumber(),
+              cashPool.toNumber(),
+              amount * 1e6,
+              target.toNumber(),
+              4,
+            );
+
+      const boughtAfterFee = (bought * FEE) / FEE_DENOM;
+      const tok = boughtAfterFee * (1 - slippage * 0.01);
+      setWithoutSlippage(Number((Math.floor(boughtAfterFee) / 1e6).toFixed(6)));
```
Comment (murbard) [inline frontend/app/src/pages/BuySell/Conversion.tsx:77]: please verify those calculations with the output of the contract

File: frontend/app/src/pages/BuySell/RemoveLiquidity.tsx [hunk 5]
```diff
@@ -49,11 +49,11 @@ const RemoveLiquidityComponent: React.FC<WithTranslation> = ({ t }) => {
 
   const calcMinValues = (slippage: number, lqtBurned: number) => {
     if (cfmmStorage) {
-      const { cashPool, tokenPool, lqtTotal } = cfmmStorage;
+      const { cashPool, tezPool, lqtTotal } = cfmmStorage;
```
Comment (murbard) [inline frontend/app/src/pages/BuySell/RemoveLiquidity.tsx:52]: check order

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx (hunk 2 above)
Comment (shekhar-shubhendu) [reply frontend/app/src/pages/BuySell/AddLiquidity.tsx:51]: yes, this is okay. it is named destructuring. the order doesn't matter.

File: frontend/app/src/pages/BuySell/AddLiquidity.tsx (hunk 1 above)
Comment (murbard) [reply frontend/app/src/pages/BuySell/AddLiquidity.tsx:54]: Ah nm this is for adding liquidity

**Pull request merged**